irony-completion.el

```diff
@@ -252,11 +268,12 @@ A candidate is composed of the following elements:
  2. The [result-]type of the candidate, if any.
  3. If non-nil, contains the Doxygen brief documentation of the
     candidate.
- 4. The signature of the candidate excluding the result-type
+ 4. The accessiblity of the candidate.
```

Comment:
It could be non-negligible, I could not say easily.
If the list of candidates is quite big I think most of the time is spent in irony-server and in the process filter.